README.md

```diff
@@ -1,24 +1,41 @@
-# devlup-labs
-
-## Project setup
-```
-npm install
-```
-
-### Compiles and hot-reloads for development
-```
-npm run serve
-```
-
-### Compiles and minifies for production
-```
-npm run build
-```
-
-### Lints and fixes files
-```
-npm run lint
-```
-
-### Customize configuration
-See [Configuration Reference](https://cli.vuejs.org/config/).
+# DevlUp Labs Portfolio
+
+Portfolio of the Open-Source Community at IIT Jodhpur.
+
+## Project Setup
+
+1. Fork this repository!
+
+2. Clone this repository and setup a remote to the original repository.
+
+3. Open up your IDE and in your terminal run the following commands:
+
+ * npm install
+
+ * Open up a second terminal and cd into the 'data' folder and run=> py -3 -m http.server --bind 0.0.0.0 5000
```

Comment:
Are we using this now? Didn't we move to sheets?

Reply:
The images are still being used from that folder